tests/EvmAcceptanceTests/test/Delegatecall.js

```diff
@@ -0,0 +1,29 @@
+const {expect} = require("chai");
+const {ethers} = require("hardhat");
+
+describe("Delegatecall functionality", function () {
+  before(async function () {
+    const Contract = await ethers.getContractFactory("Delegatecall");
+    this.delegateContract = await Contract.deploy();
+
+    const TestContract = await ethers.getContractFactory("TestDelegatecall");
+    this.testDelegateContract = await TestContract.deploy();
+  });
+
+  it("should delegate function call correctly", async function () {
+    const VALUE = 1000;
```

Comment:
This value in Wei will get lost in Zilliqa due to the fact that we round to 1,000,000. 
You need to use a value larger than 1mln.

Reply:
yep it worked with 1,000,000